pandas/tests/reshape/test_pivot.py

```diff
@@ -516,6 +516,7 @@ def test_pivot_index_with_nan(self, method):
             result = pd.pivot(df, "b", "a", "c")
         tm.assert_frame_equal(result, pv.T)
 
+    @pytest.mark.filterwarnings("ignore:Timestamp.freq is deprecated:FutureWarning")
```

Comment:
Same here

Reply:
looks like in all three cases they no longer are needed.  will remove in my next follow-up branch